src/api/firebase.js

```diff
@@ -1,5 +1,5 @@
-import { collection, onSnapshot } from 'firebase/firestore';
 import { db } from './config';
+import { collection, onSnapshot, addDoc } from 'firebase/firestore';
```

Comment:
@DwightTheShark @tinazarb Did you figure out to add `addDoc` to this file based on Firebase documentation?